### PR TITLE
Great People

### PR DESCRIPTION
**Pull request by MeuhV1**

### CODE REVIEW

File: Assets/Python/GreatPeople.py [hunk 1]
```diff
@@ -749,7 +749,7 @@ def setup():
 		"Tewodros", # 19th 
 		"Menelik", # 19th 
 		iModern,
-		"Mengistu Haile Mariam", # 20th 
+		u"Mengistu Haile Mariam", # 20th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: Technically that does not need to be unicode.

File: Assets/Python/GreatPeople.py [hunk 2]
```diff
@@ -2042,11 +2045,20 @@ def setup():
 		"Pieter Stuyvesant", # 17th 
 		"Jan van Riebeeck", # 17th 
 		"Jan Coen", # 17th 
+		iIndustrial,
+		"Brenninkmeijer", # 19th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: I prefer full names, so please choose one first name as a representative GP, like I did with the Wright brothers.

**Comment by dguenms** [comment]
Ready to merge after the comment is incorporated.

File: Assets/Python/GreatPeople.py [hunk 3]
```diff
@@ -2046,7 +2046,7 @@ def setup():
 		"Jan van Riebeeck", # 17th 
 		"Jan Coen", # 17th 
 		iIndustrial,
-		"Brenninkmeijer", # 19th 
+		"Brenninkmeijer Brothers", # 19th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: It's one GP unfortunately, you have to pick one of the brothers.

File: Assets/Python/GreatPeople.py [hunk 4]
```diff
@@ -2042,11 +2045,20 @@ def setup():
 		"Pieter Stuyvesant", # 17th 
 		"Jan van Riebeeck", # 17th 
 		"Jan Coen", # 17th 
+		iIndustrial,
+		"Brenninkmeijer Brothers", # 19th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: It's one GP unfortunately, you have to pick one of the brothers.

**Comment by MeuhV1** [comment]
Then the Wright Brother should also be fixed. I used them as example.

**Comment by dguenms** [comment]
I thought I chose one there ...? Sorry for giving them as an example then.

**Pull request merged**